src/coreclr/vm/gdbjit.cpp

```diff
@@ -1384,22 +1384,23 @@ bool FunctionMember::GetBlockInNativeCode(int blockILOffset, int blockILLen, TAD
 
 void FunctionMember::DumpTryCatchBlock(char* ptr, int& offset, int ilOffset, int ilLen, int abbrev)
 {
-    TADDR startOffset;
-    TADDR endOffset;
+    TADDR* startOffset;
+    TADDR* endOffset;
 
-    if (!GetBlockInNativeCode(ilOffset, ilLen, &startOffset, &endOffset))
+    if (!GetBlockInNativeCode(ilOffset, ilLen, startOffset, endOffset))
         return;
```

Comment:
This doesn't look correct. These pointers aren't initialized and are being passed to a function that is dereferencing them. They are also being dereferenced below.